src/bin/cql-stress-cassandra-stress/operation/mixed.rs

```diff
@@ -111,6 +148,18 @@ impl MixedOperationFactory {
             stats,
         })
     }
+
+    async fn conditional_create_factory<Factory, Fut: Future<Output = Factory>>(
+        ratios: &OperationRatio,
+        command_kind: &MixedSubcommand,
+        create_factory_fut: impl FnOnce() -> Fut,
+    ) -> Option<Factory> {
+        if ratios.contains(command_kind) {
+            Some(create_factory_fut().await)
+        } else {
+            None
+        }
+    }
```

Comment:
In instantiations of this functions, is `Factory` equal to `Result<impl CassandraStressOperation>`? I struggled to find where this result comes from in outer code, but here it is. Maybe let's constrain the function further by imposing constraints on `Factory` to be `Result<impl CassandraStressOperation>`?